File: backend-C#/DoctorsOffice/DbModels/Schedule.cs
using System;
using System.Text.Json.Serialization;

namespace DoctorsOffice.DbModels
{
    public class Schedule
    {
        public int Id { get; set; }
        
        public DateTime Date { get; set; }
        public DateTime StartTime { get; set; }
        public DateTime EndTime { get; set; }
        
        [JsonIgnore]
        public Doctor Doctor { get; set; }

    }
}

File: backend-Java/src/main/java/pl/damkac/schedule/Schedule.java
package pl.damkac.schedule;


import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;
import pl.damkac.doctor.Doctor;

import javax.persistence.*;
import java.time.LocalDate;
import java.time.LocalDateTime;

@Entity
@Data
@AllArgsConstructor
@NoArgsConstructor
public class Schedule {

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long id;

    private LocalDate date;
    private LocalDateTime startTime;
    private LocalDateTime endTime;

    @ManyToOne(cascade = CascadeType.ALL)
    @JoinColumn(name = "doctor_id")
    private Doctor doctor;
}
